server/globals.ts

```diff
@@ -16,3 +16,48 @@ declare let IPTools: typeof import('./ip-tools').IPTools;
 declare let Sockets: typeof import('./sockets');
 // let TeamValidator: typeof import('../sim/team-validator');
 declare let TeamValidatorAsync: typeof import('./team-validator-async');
+
+interface Group {
+	symbol: string,
+	id: string,
```

Comment:
Should `id` be an `ID` type? (is it always all lowercase etc?)

Reply:
Yes, I think group IDs are guaranteed to be IDs. However, having to type `/** @type {ID} */ ('moderator')` might be too annoying in config.js, if that's the plan.

Reply:
True. Is `config.js` even included in typechecking though? Its not in `tsconfig.json`'s `include`, but maybe it still gets pulled in?

Reply:
I'm not sure. It _was_, but at some point this was changed, I think.